src/connection.cpp

```diff
@@ -1200,6 +1213,9 @@ namespace oxen::quic
             tls_creds{context->tls_creds},
             di{*this}
     {
+        conn_established_cb = (context->conn_established_cb) ? std::move(context->conn_established_cb) : nullptr;
```

Comment:
moving this (and the next one) out of the IOContext will result in only the first connection a listener makes getting the callbacks.

Reply:
That is entirely the point for storage server ping testing